DTS-STN/Alpha-Site
# Report a problem fix

# Description

[Fix Report a Problem so that when in French, checkbox labels do not overlap](https://trello.com/c/2tTuezHw/398-fix-report-a-problem-so-that-when-in-french-checkbox-labels-do-not-overlap)
[Fix repeated string in Report a Problem "Provide more details" prompt](https://trello.com/c/GgAwnzKG/401-fix-repeated-string-in-report-a-problem-provide-more-details-prompt)

I removed the duplicate optional element in the report a problem dropdown, and I spaced out the checkboxes on the mobile view as well. I also adjusted the checkboxes in the signup form for the minority group, since some text also overlapped on small screens.

![image](https://user-images.githubusercontent.com/72703030/126528667-c5c5b058-63ba-4560-b7f5-806120e8bceb.png)
![image](https://user-images.githubusercontent.com/72703030/126528751-5be47816-86b2-4f22-8483-06e95b2ff52e.png)
![image](https://user-images.githubusercontent.com/72703030/126528831-61f02a47-579d-4c9b-a2cb-e8782f4af0f9.png)

## Acceptance Criteria

The report a problem dropdown shouldn't have two optional texts for more details.
The checkboxes labels shouldn't overlap each other on all screen sizes.
The checkboxes labels for choosing the minority group in the signup form also shouldn't overlap.

## Test Instructions

1. Test the report a problem dropdown on different screen sizes
2. Go to the signup page and click on yes for "Do you identify as a member of a visible minority in Canada?"
3. Test the checkboxes on different screen widths and make sure the text doesn't overlap
4. Test in english and french

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
